README.md

```diff
@@ -157,8 +157,7 @@ For more information on our configuration manager visit its readme [here](https:
 
 - `SAS_EXPRESS_BIND_HOST`: address on which the server will be listening, defaults to `127.0.0.1`.
 - `SAS_EXPRESS_PORT`: port on which the server will be listening, defaults to `8080`.
-- `SAS_EXPRESS_LOG_MODE`: enable console logging of "all" HTTP requests, only "errors", or nothing by
```

Comment:
Is `SAS_EXPRESS_LOG_MODE` being removed as a valid configuration value?

Reply:
Funny, but also weirdly enough it's a very old legacy mode which has been removed for a while now.